exchanges/ftx/ftx_websocket_test.go

```diff
@@ -0,0 +1,209 @@
+package ftx
```

Comment:
Please move over the other websocket tests from ftx_test.go eg `TestParsingWSOBData`